podpac/core/coordinates/test/test_polar_coordinates.py

```diff
@@ -1,261 +1,261 @@
+
+#         with pytest.raises(KeyError, match="Cannot get dimension"):
+#             c["other"]
+
+#     def test_get_index_slices(self):
+#         c = PolarCoordinates(center=[1.5, 2.0], radius=[1, 2, 4, 5, 6], theta_size=8, dims=["lat", "lon"])
+
+#         # full
+#         c2 = c[1:4, 2:4]
+#         assert isinstance(c2, PolarCoordinates)
+#         assert c2.shape == (3, 2)
+#         assert_allclose(c2.center, c.center)
+#         assert c2.radius == c.radius[1:4]
+#         assert c2.theta == c.theta[2:4]
+#         assert c2.dims == c.dims
+#         assert_allclose(c2.coordinates[0], c.coordinates[0][1:4, 2:4])
+#         assert_allclose(c2.coordinates[1], c.coordinates[1][1:4, 2:4])
+
+#         # partial/implicit
+#         c2 = c[1:4]
+#         assert isinstance(c2, PolarCoordinates)
+#         assert c2.shape == (3, 8)
+#         assert_allclose(c2.center, c.center)
+#         assert c2.radius == c.radius[1:4]
+#         assert c2.theta == c.theta
+#         assert c2.dims == c.dims
+#         assert_allclose(c2.coordinates[0], c.coordinates[0][1:4])
+#         assert_allclose(c2.coordinates[1], c.coordinates[1][1:4])
+
+#         # stepped
+#         c2 = c[1:4:2, 2:4]
+#         assert isinstance(c2, PolarCoordinates)
+#         assert c2.shape == (2, 2)
+#         assert_allclose(c2.center, c.center)
+#         assert c2.radius == c.radius[1:4:2]
+#         assert c2.theta == c.theta[2:4]
+#         assert c2.dims == c.dims
+#         assert_allclose(c2.coordinates[0], c.coordinates[0][1:4:2, 2:4])
+#         assert_allclose(c2.coordinates[1], c.coordinates[1][1:4:2, 2:4])
+
+#         # reversed
+#         c2 = c[4:1:-1, 2:4]
+#         assert isinstance(c2, PolarCoordinates)
+#         assert c2.shape == (3, 2)
+#         assert_allclose(c2.center, c.center)
+#         assert c2.radius == c.radius[4:1:-1]
+#         assert c2.theta == c.theta[2:4]
+#         assert c2.dims == c.dims
+#         assert_allclose(c2.coordinates[0], c.coordinates[0][4:1:-1, 2:4])
+#         assert_allclose(c2.coordinates[1], c.coordinates[1][4:1:-1, 2:4])
+
+#     def test_get_index_fallback(self):
+#         c = PolarCoordinates(center=[1.5, 2.0], radius=[1, 2, 4, 5], theta_size=8, dims=["lat", "lon"])
+#         lat, lon = c.coordinates
+
+#         Ra = [3, 1]
+#         Th = slice(1, 4)
+#         B = lat > 0.5
+
+#         # int/slice/indices
+#         c2 = c[Ra, Th]
+#         assert isinstance(c2, DependentCoordinates)
+#         assert c2.shape == (2, 3)
+#         assert c2.dims == c.dims
+#         assert_equal(c2["lat"].coordinates, lat[Ra, Th])
+#         assert_equal(c2["lon"].coordinates, lon[Ra, Th])
+
+#         # boolean
+#         c2 = c[B]
+#         assert isinstance(c2, StackedCoordinates)
+#         assert c2.shape == (22,)
+#         assert c2.dims == c.dims
+#         assert_equal(c2["lat"].coordinates, lat[B])
+#         assert_equal(c2["lon"].coordinates, lon[B])
```

Comment:
Why  not just delete the file?

Reply:
Ah I overlooked polar coordinates. I should actually update the polar coordinates and then reinstate the tests.